packages/abstraxion/src/hooks/useAbstraxionSigningClient.ts

```diff
@@ -43,14 +47,12 @@ export const useAbstraxionSigningClient = () => {
 
         setAbstractClient(directClient);
       } catch (error) {
-        console.log("Something went wrong: ", error);
+        setAbstractClient(undefined);
```

Comment:
We probably still want to log something here, no? This is not expected behavior, right?

Reply:
Yea I agree. We'll probably end up implementing some sort of banner/toast notification system at some point but a console log should do for now. I'll get that in